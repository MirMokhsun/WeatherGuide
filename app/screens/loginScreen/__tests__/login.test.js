import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import LoginScreen from '..';

describe('LoginScreen', () => {
    test('LoginScreen snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<LoginScreen />);
        expect(snapshot).toMatchSnapshot();
    });
});