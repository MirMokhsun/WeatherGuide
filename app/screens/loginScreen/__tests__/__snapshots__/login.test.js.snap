// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Login snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#45d679",
    }
  }
>
  <Button
    onPress={[Function]}
    title="loginScreen"
  />
  <Button
    onPress={[Function]}
    title="Main"
  />
</View>
`;
